core/examples/http_bench_json_ops.js

```diff
@@ -23,16 +24,12 @@ async function serve(rid) {
   try {
     while (true) {
       await Deno.core.read(rid, requestBuf);
-      await Deno.core.write(rid, responseBuf);
+      const nwritten = ops.op_try_write_sync(rid, responseBuf);
+      if (nwritten !== responseBuf.byteLength) {
+        await Deno.core.write(rid, responseBuf.subarray(nwritten));
+      }
```

Comment:
Can you move this code into a separate function to better represent what typical user code would look like? I wonder if it will slow down.